### PR TITLE
chore: minor improvements in code style

### PR DESCRIPTION
**Pull request by andrii-zakurenyi**

### CODE REVIEW

**Comment by sheldonhull** [comment]
@andrii-zakurenyi flipped to draft. That will prevent noise, actions, and merge until you mark as ready for review. cheers

**Comment by sheldonhull** [comment]
@andrii-zakurenyi you'll want to rebase on main (`git town sync`) fyi due to additions of github workflows, linting and such from #9. Cheers

**Comment by sheldonhull** [comment]
Nice! Can you enable automerge and customize the squashed message to just the valid changes? I'll get it approved right away as I'm looking.

File: vault/vault.go [hunk 1]
```diff
@@ -168,32 +167,24 @@ func (v Vault) getAccessToken() (string, error) {
...
-	grant := struct {
-		AccessToken, TokenType string
-		ExpiresIn              int
-		// TODO cache the grant until it expires
-	}{}
-
-	if err = json.Unmarshal(response, &grant); err != nil {
-		log.Print("[INFO] parsing grant response:", err)
-		return "", err
+	// TODO: cache the token until it expires.
```
Comment (sheldonhull) [inline vault/vault.go:181]: @andrii-zakurenyi is this still applicable? If so we should create a github issue or other item to track this. mark this as closed after that or removing if appropriate.

**Review by sheldonhull — CHANGES_REQUESTED** [review]
Looks great overall, just double check what I put in there.
Please enable auto-squash with the edited body of what you changed in a couple bullets and then I'll get this merged. Cheers.

File: vault/vault.go (hunk 1 above)
Comment (andrii-zakurenyi) [reply vault/vault.go:181]: Yes, it still is. Created a PBI in the backlog

**Review by sheldonhull — APPROVED** [review]
✨

**Pull request merged**